pkg/ovs/openflow/interfaces.go

```diff
@@ -62,6 +62,7 @@ const (
 	NxmFieldARPOp       = "NXM_OF_ARP_OP"
 	NxmFieldReg         = "NXM_NX_REG"
 	NxmFieldTunMetadata = "NXM_NX_TUN_METADATA"
+	NxmFieldIPTos       = "NXM_OF_IP_TOS"
```

Comment:
I saw you used both "Tos" and "ToS". I would suggest "ToS" (so NxmFieldIPToS) or "TOS".

Reply:
Renamed to "NxmFieldIPToS"